qcelemental/tests/test_constants.py

```diff
@@ -78,6 +78,7 @@ def test_c_header():
     os.remove("header.h")
 
 
+@pytest.mark.xfail(True, reason='comparison data not available for installed repository', run=True, strict=False)
```

Comment:
Should we check if these files are there or not?

Reply:
I guess if it is `xfail` its fine. No reason to do anything complex.

Reply:
Yeah, it'll xpass if source, xfail if installed. Agree there's no need to make tests a test of where it's running from. The conda pkg is `v0.1.0a` + this PR.